server/middleware/file.js

```diff
@@ -14,7 +15,7 @@ const storage = multerS3({
     s3,
     bucket: BUCKET_NAME,
     metadata: (req, file, callback) => {
-        callback(null, { originalName: file.originalname });
+        callback(null, { originalName: querystring.escape(file.originalname) });
```

Comment:
Looks like the [docs](https://nodejs.org/api/querystring.html#querystring_querystring_escape_str) say that you generally shouldn't use `.escape()` directly and should use `.stringify()` instead.

Reply:
I did look at that, but `.stringify() is for objects, so I think `escape()` is more specific and appropriate for our use case?

Reply:
I'm leaning to agree with Douglas on this - the documentation states it should be used instead as escape.

Reply:
If I was to use it, I would have to wrap the string in a object just to satisfy `stringify()` which uses escape underneath anyway.